sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -198,10 +191,8 @@
                                     />
                                     <field
                                         name="price_unit"
-                                        attrs="{
-                                            'required': [('display_type', '=', False)],
-                                            'invisible': [('display_type', '=', True)],
-                                        }"
+                                        required="display_type == False"
+                                        invisible="display_type == True"
```

Comment:
Please use `column_invisible` see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0